src/Sylius/Bundle/CoreBundle/Form/Extension/CartItemTypeExtension.php

```diff
@@ -70,4 +70,9 @@ public function getExtendedType(): string
     {
         return CartItemType::class;
     }
+
+    public static function getExtendedTypes(): iterable
+    {
+        return [CartItemType::class];
```

Comment:
What would you say for:
```suggestion
        return [$this->getExtendedType()];
```
or
```suggestion
        yield $this->getExtendedType();
```
to reuse still existing method?

Reply:
@lchrusciel Wrong button... this cannot be done like this, cause new method is `static`, thus it's not called on object (no `$this` etc.), but statically like `ClassType:: getExtendedType()`.

Reply:
You are right 👍